test/metabase/db/load_and_dump_test.clj

```diff
@@ -0,0 +1,51 @@
+(ns metabase.db.load-and-dump-test
```

Comment:
Test namespaces should be `ORIGINAL-NAMESPACE-test` e.g. `metabase.cmd.load-from-h2-test`. Tools like CIDER know that namespaces are associated with one another. 

For places that don't really have a corresponding match you should still try to put them in the same folder so I would name this `metabase.cmd.load-and-dump-test`